Fix the install path for config file
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -482,8 +482,8 @@
 
 
 install-sudosh.conf:
-	test -z "$(sysconfdir)" || $(mkdir_p) "$(DESTDIR)$(sysconfdir)"
-	test -f $(sysconfdir)/sudosh.conf || $(INSTALL) -o 0 -g 0 -m 0444 '$(srcdir)/sudosh.conf' '$(sysconfdir)/sudosh.conf'
+	test -z "$(DESTDIR)$(sysconfdir)" || $(mkdir_p) "$(DESTDIR)$(sysconfdir)"
+	test -f "$(DESTDIR)$(sysconfdir)/sudosh.conf" || $(INSTALL) -o 0 -g 0 -m 0444 '$(srcdir)/sudosh.conf' '$(DESTDIR)$(sysconfdir)/sudosh.conf'
 
 install: install-am install-sudosh.conf
 
